bundles/org.openhab.binding.velbus/README.md

```diff
@@ -2,139 +2,232 @@
+| vmbmeteo    | Weather station with thermometer, anemometer, rain sensor and light sensor                 |
+| vmbpirc     | Motion and twilight sensor for ceiling mounting                                            |
+| vmbpirm     | Mini motion and twilight sensor for recessed or surface mounting                           |
+| vmbpiro     | Outdoor motion, twilight and temperature sensor, Theben                                    |
+| vmbrfr8s    | 8 channel RF receiver module                                                               |
+| vmbvp1      | Doorbird interface module                                                                  |
 
 ## Discovery
 
-The Velbus bridge cannot be discovered automatically.
-It has to be added manually by defining the serial port of the Velbus Configuration module for the Velbus Serial Bridge or by defining the IP Address and port for the Velbus Network Bridge.
+The Velbus bridge cannot be discovered automatically. Configure it manually by defining the serial port of the Velbus Configuration module for the Velbus Serial Bridge or by defining the IP address and port for the Velbus Network Bridge, as described in the [`Thing Configuration`](#thing-configuration) section.
 
-Once the bridge has been added as a thing, a manual scan can be launched to discover all other supported Velbus devices on the bus.
-These devices will be available in the inbox.
-The discovery scan will also retrieve the channel names of the Velbus devices.
+Once the bridge has been configured with openHAB, a manual scan can be initiated to discover Velbus modules with an assigned address. Addresses can be assigned via the official configuration software, and is a required step before a Velbus installation can work correctly.
+
+The discovery scan can take a few minutes to complete. Modules discovered during this scan will appear in the inbox. This procedure will also retrieve the channel names of the Velbus devices.
 
 ## Thing Configuration
 
 The Velbus bridge needs to be added first.
 
+### Velbus Serial Bridge
+
 For the Velbus Serial Bridge it is necessary to specify the serial port device used for communication.
-On Linux systems, this will usually be either `/dev/ttyS0`, `/dev/ttyUSB0` or `/dev/ttyACM0` (or a higher  number than `0` if multiple devices are present).
-On Windows it will be `COM1`, `COM2`, etc.
 
-In the things file, this might look e.g. like
+On Linux and other UNIX systems, it is recommended to use a more stable symbolic device path such as `/dev/serial/by-id/usb-Velleman_Projects_VMB1USB_Velbus_USB_interface-if00`, as it will always refer at to a Velbus configuration module, and not an arbitrary serial device. If this is not a concern or an option, it is valid to refer to the serial device directly with a path such as `/dev/ttyS0`, `/dev/ttyUSB0` or `/dev/ttyACM0` (or a number other than `0` if multiple serial devices are connected.)
```

Comment:
```suggestion
On Linux and other UNIX systems, it is recommended to use a more stable symbolic device path such as `/dev/serial/by-id/usb-Velleman_Projects_VMB1USB_Velbus_USB_interface-if00`, as it will always refer at to a Velbus configuration module, and not an arbitrary serial device.
If this is not a concern or an option, it is valid to refer to the serial device directly with a path such as `/dev/ttyS0`, `/dev/ttyUSB0` or `/dev/ttyACM0` (or a number other than `0` if multiple serial devices are connected.)
```